cluster-provision/base/scripts/dnsmasq.sh

```diff
@@ -28,4 +30,4 @@ iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
 iptables -A FORWARD -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
 iptables -A FORWARD -i br0 -o eth0 -j ACCEPT
 
-exec dnsmasq -d ${DHCP_HOSTS} --dhcp-range=192.168.66.10,192.168.66.200,infinite
+exec dnsmasq -d ${DHCP_HOSTS} --dhcp-range=192.168.66.10,192.168.66.200,infinite --dhcp-range=::,static
```

Comment:
Sorry I missed this originally, but the range should be from fd00/8 too, shouldn't it? With the exception of ::1.

Reply:
This configures a "static" dhcp configuration. That means that it will only offer IPs to requests where the mac is  specified in ${DHCP_HOSTS}.

Reply:
Oh, my bad, I missed the for cycle. Thanks for the explanation.

Reply:
But that's for the primary nics, aren't we going to set ipv6 addresses at secondary nics ?

Reply:
Regarding the configuration here, I am afraid it is a bit more complex than this one liner.
See dnsmasq configuration from the nmstate project: https://github.com/nmstate/nmstate/blob/2cf0bb8812415a96b0f35d1f0d4e4ef12946ea6d/tests/integration/dynamic_ip_test.py#L85

Reply:
Can very well be. I can only confirm that if the IPs and the segments work, the VMs can communicate with each other.